io.github.zaps166.QMPlay2.Devel.json

```diff
@@ -0,0 +1,169 @@
+{
+  "app-id": "io.github.zaps166.QMPlay2",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.6",
+  "sdk": "org.kde.Sdk",
+  "add-extensions": {
+      "org.freedesktop.Platform.ffmpeg-full": {
+          "directory": "lib/ffmpeg",
+          "version": "23.08",
+          "add-ld-path": ".",
+          "autodelete": false
+      }
+  },
+  "cleanup-commands": [
+      "mkdir -p /app/lib/ffmpeg"
+  ],
+  "command": "QMPlay2",
+  "rename-icon": "QMPlay2",
+  "rename-desktop-file": "QMPlay2.desktop",
+  "rename-appdata-file": "QMPlay2.appdata.xml",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--filesystem=home",
+    "--filesystem=/run/media",
+    "--filesystem=/media",
```

Comment:
`/mnt` too?

Reply:
Kodi use it, we can add if it's necessary. 

https://github.com/flathub/tv.kodi.Kodi/blob/master/tv.kodi.Kodi.yml#L12-L31

Reply:
Yes, please.